### DISCUSSION TITLE
It appears Apple have revised their new App Store rules for NFTs and cryptocurrency

### DISCUSSION THREAD

**Discussion by bapex** [Web3Auth/Web3Auth#762] [Help]
It appears Apple have revised their new App Store rules for NFTs and cryptocurrency.
https://developer.apple.com/news/?id=xk8d7p8c

> Revised 3.1.1: “Apps may not use their own mechanisms to unlock content or functionality, such as license keys, augmented reality markers, QR codes, cryptocurrencies and cryptocurrency wallets, etc.”

In this case, can we use torus login and wallet for ios app?

**Comment by chaitanyapotti** [comment]
we have a couple of apps in prod ([rove tickets](https://roveworld.xyz/) etc. using web3auth logins in app store.
we will have to wait and get feedback from apple store team as to what these new changes mean to us
